docs/stylesheets/base/utilities.css

```diff
@@ -12,3 +12,13 @@
   gap: 20px;
   padding: 10rem;
 }
+
+.showcase-badge {
+  background-color: var(--secondary-400);
+  padding: 2.6rem;

```

Comment:
You can use 3-4 variables for paddings and other spacings of different values, as was suggested in the style guide. This way all the spacings across your website will be consistent.

Reply:
Yeah for now I have hard-coded the value. Implement this in the coming release.